### DISCUSSION TITLE
Allow widgets to resize to 1x1

### DISCUSSION THREAD

**Discussion by ngelderloos** [iSoron/uhabits#797] [Help & FAQ]
I am unable to resize most widgets to 1x1 size. I was able to do this on my Galaxy S6, but can no longer do this on Galaxy S20 FE. It would be helpful to be able to resize to whatever size fits my screen/reading ability. Was this feature recently removed, or is there something on my phone that is preventing this ability?

**Comment by iSoron** [comment]
> I am unable to resize most widgets to 1x1 size

Hi @ngelderloos, only the checkmark widget can be resized to 1x1; all the remaining widgets require more space. There was no change in our code, but perhaps Samsung has now decided to enforce these constraints. If you want more control over the widgets, I would suggest switching to Nova Launcher.

**Reply by ngelderloos** [reply]
I am curious why the remaining widgets require more space? I understand that some people may have difficultly viewing the details when resized to 1x1, but I have had no difficultly over the past 3+ years viewing them at this size. Would it be possible to allow the widget to be resized to whatever size the user prefers, maybe defaulting to 1x2, but allowing the size to be 1x1?
